### ISSUE TITLE
No BorderRadius Control ?

### ISSUE THREAD

**Issue by marcojr** [suragch/audio_video_progress_bar#20]
on the examples at https://github.com/suragch/audio_video_progress_bar , the progress bar has no border radius

<img width="314" alt="deep_purple_theme" src="https://user-images.githubusercontent.com/856617/128789980-519b595f-a630-4007-bfeb-b64d6fcb9ed1.png">

however when executing, take a look...

<img width="324" alt="Screenshot 2021-08-10 at 01 16 48" src="https://user-images.githubusercontent.com/856617/128789998-36700679-cd05-4558-a6a2-bf7b8fb90893.png">

It places the border of the progress bar with radius. Is there some how to control it ? I can't find anything on the doc.

I really wish to set my progress bar as it seens on the github pag, I mean: borderRadius = 0 on progress bar.

**Comment by suragch** [comment]
Yes, that used to be the default and I haven't updated the pictures yet.

Currently there is no way to change whether the bar is round or not. I'll leave this issue open and if enough people upvote it then I'll add an option for it.

The easiest way for you to solve your problem right now, though, is to just copy the source code to your project and delete the following line from the `_drawBar` method:

```dart
..strokeCap = StrokeCap.round
```

**Comment by enseitankad0** [comment]
I will be grateful if you could implement this enhancement. Thanks in advance!

**Comment by marcojr** [comment]
thanks, @suragch  ! Yes, it could be nice to allow users to choose ;)

**Comment by suragch** [comment]
Version 0.9.0 adds this functionality.

```dart
barCapShape: BarCapShape.square,
```

```dart
/// The shape of the progress bar at the left and right ends.
enum BarCapShape {
  /// The left and right ends of the bar are round.
  round,

  /// The left and right ends of the bar are square.
  square,
}
```

If you find any problems with it feel free to reopen this issue or create a new one.

**Comment by enseitankad0** [comment]
Thank you very much!
